.github/workflows/release_workflow.yml

```
@@ -17,6 +18,22 @@ jobs:
       publish_prerelease: true
       changelog_max_issues: 100
 
+  notify:
+    if: github.event.pull_request.merged == true
+    needs: publish_alpha
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)